contents/tutorials/intercom-session-replays.md

```diff
@@ -0,0 +1,238 @@
+---
+title: How to add session replays to Intercom
+date: 2024-02-13
+author: ["ian-vanagas"]
+tags: ['session replay']
+---
+
+Session replays are a useful support tool for debugging and recreating issues. The errors, console, and network data along with the rest of PostHog's tools make it a powerful support platform.
+
+To get easy access to session replays in Intercom, you can add them as either user data or events. To show you how to do this, we build a basic Next.js app, add PostHog, add Intercom, and set up the session replay link to show in both places in Intercom.
+
+## 1. Create a basic Next.js app
+
+First, make sure [Node is installed](https://nodejs.dev/en/learn/how-to-install-nodejs/) (18.17 or newer), and then create a Next.js app:
+
+```bash
+npx create-next-app@latest intercom-sessions
+```
+
+Select **No** for TypeScript, **Yes** for `use app router`, and the defaults for every other option. Once created, go into your `app/page.js` file and add a basic page with a spot for users to enter their email.
+
+```js
+// app/page.js
+"use client"
+import { useState } from 'react';
+
+export default function Home() {
+	const [email, setEmail] = useState('');
+
+  return (
+     <main>
+      <h1>Hello Intercom!</h1>
+      <h3>Enter your email:</h3>
+      <input 
+        value={email} 
+        onChange={(e) => setEmail(e.target.value)} 
+      />
+    </main>
+  );
+}
+
+```
+
+Once done, run `npm run dev` and go `http://localhost:3000`  to see your app.
+
+![App running](../images/tutorials/intercom-session-replays/app.png)
+
+## 2. Add PostHog
+
+To set up session replays, we install PostHog in our app. If you don't have a PostHog instance, you can [sign up for free](https://us.posthog.com/signup).
+
+> **Important:** Make sure to enable "Record user sessions" in [your project settings](https://us.posthog.com/project/settings/project-replay).
+
+After doing this, install the [`posthog-js`](/docs/libraries/js) SDK:
+
+```bash
+npm i posthog-js
+```
+
+Next, create a `providers.js`  file in your `app`  folder. In it, initialize PostHog with your project API key and instance address (from [your project settings](https://us.posthog.com/project/settings/project-details)) and export a provider component. If you use a US Cloud instance, also make sure your `ui_host` config option is set to `https://us.posthog.com`.
```

Comment:
Same re the link here. It takes me to a page not found, but this URL works for me: https://us.posthog.com/settings/project#variables

Reply:
Hmm its probably worth checking other URLs like this are working (in the other tutorials/docs)

Reply:
Checked a bunch of them that include `https://us.posthog.com/project/` and they work fine.